code/datums/mutantraces.dm

```diff
@@ -1250,6 +1307,12 @@
 	icon_state = "roach"
 	icon_override_static = 1
 	override_attack = 0
+	tail = ROACH_TAIL
+
+	New()
+		..()
+		set_tail(mob, src.tail)
+		mob.blood_color = "#AEB555"
```

Comment:
better to do this as/in a separate pr - it doesn't really relate to the rest

Reply:
Will do!